platform/broadcom/sonic-platform-modules-cel/services/fancontrol/fancontrol

```diff
@@ -158,6 +161,23 @@ function LoadConfig
 		let fcvcount=fcvcount+1
 	done
 	echo
+
+
+	let tscount=0
+	for ts in $MAXTEMPCRIT
+	do
+		CSTEMP[$tscount]=`echo $ts | cut -d '=' -f1`
```

Comment:
This file is using tabs instead of spaces. Can you please replace all tabs with four spaces?

Reply:
done in https://github.com/Azure/sonic-buildimage/pull/4678/commits/cb7def85f22cba21f48c656eda21b81a578d28ec